src/app/zap-templates/zcl/data-model/chip/operational-credentials-cluster.xml

```diff
@@ -35,6 +35,7 @@ limitations under the License.
     <item name="InvalidNOC" value="0x03"/>
     <item name="MissingCsr" value="0x04"/>
     <item name="TableFull" value="0x05"/>
+    <item name="InvalidAdminSubject" value="0x06"/>
```

Comment:
Where do we return this value?  I don't think we do, though the spec requires us to...